src/app-layer-htp.c

```diff
@@ -374,6 +374,8 @@ void HTPStateFree(void *state)
         htp_connp_destroy_all(s->connp);
     }
 
+    HttpRangeFreeBlock(s->file_range);
```

Comment:
Should we not try to keep this buffer even if it was incomplete ?
That means calling `HTPFileCloseHandleRange`

Reply:
`s->file_range` contains the flow specific data, so it needs to be freed with the state. Normally if we'd get here we would have called the "close" logic before it, so that would be responsible for handling any data that is still useful somehow.